packages/identity-service/src/authMiddleware.js

```diff
@@ -38,6 +39,53 @@ const queryDiscprovForUserId = async (walletAddress, handle) => {
   }
 }
 
+/**
+ * queryDiscprovForUserId - Queries the discovery provider for the user w/ the walletaddress
+ * @param {number} managerWalletAddress
+ * @param {number} actingUserId
+ * @param {number} authHeaders authentication headers from the manager account
+ * @returns boolean whether or not the managagingUserId manages actingUserId
+ */
+const getManagedUser = async (
+  managerWalletAddress,
+  actingUserId,
+  authHeaders
+) => {
+  const { discoveryProvider } = audiusLibsWrapper.getAudiusLibs()
+
+  const response = await axios({
+    method: 'get',
+    url: `${
```

Comment:
Just FYI I believe by default this will return pending managers as well. There's a ticket to allow filtering on the endpoint. So the options are either:
* Implement the filtering first and then update this to pass a query string (preferred)
* Update this to filter out any managers where `is_approved === null`